src/path.cpp

```diff
@@ -1,461 +1,461 @@
-/*
-	Path handling utilities.
-*/
-#if !defined(GB_SYSTEM_WINDOWS)
-#include <unistd.h>
-#endif

```

Comment:
What happened with this file? What's up with the diff?

Reply:
only one `#if defined(GB_SYSTEM_HAIKU)` for `dirent.h` was added, everything else is CRLF conversion